clients/tabby-agent/src/TabbyAgent.ts

```diff
@@ -393,6 +395,7 @@ export class TabbyAgent extends EventEmitter implements Agent {
       }
     }
     await this.anonymousUsageLogger.init({ dataStore: this.dataStore });
+    this.clientInfoStr = this.clientInfoToString(undefined);
```

Comment:
```suggestion
    this.clientInfoStr = this.clientInfoToString(options.clientProperties?.session);
```
This should combine L398 and L410.